### ISSUE TITLE
Add collections in TBX export

### ISSUE THREAD

**Issue by Federgraf** [gaois/terminologue#128]
Hello,
Would it be possible to include collections in the TBX export?
Thank you!
Best regards,

**Comment by michmech** [comment]
Yes, I suppose it would. What should they be exported as? As `<descrip type="collection">` or something like that?

**Comment by Federgraf** [comment]
Yes, that seems right. Thanks a lot!

**Comment by Federgraf** [comment]
Hello,
Could you please give me an update on this request?
Thanks!
Best,

**Comment by michmech** [comment]
Sorry, no action yet. Hopefully we'll get a chance to do a dev sprint on Terminologue sometime in the new year (2025). If yes, then this issue will definitely be included.
